### DISCUSSION TITLE
Add mobile touch support (create file open file etc...)to the Android port

### DISCUSSION THREAD

**Discussion by Yasserpro312** [godotengine/godot-proposals#4414] [Engine Core]
Add mobile touch support (create file open file etc...)to the Android port  without having to use mouse and keyboard cuz some devices have otg support but its disabled by the software (android ui like emui)

**Comment by Calinou** [comment]
Related to https://github.com/godotengine/godot/issues/59938. This is planned, but it'll likely take a while to materialize.
